packages/IMAPClient-Core.package/ICAddressBook.class/instance/delete..st

```diff
@@ -1,5 +0,0 @@
-crud
-delete: anEntry
```

Comment:
Why do you use a wrapper function for adding but not for deleting emails? 
Right now the AdressbookDialog operates directly on the collection interface for deleting.
You could use deleteEmail to encapsulate what the AddressbookDialog needs to assume about the AddressBook.

Reply:
Premature abstraction since we only want to remove an element from the collection. Nothing more is happening right now since the UI is also only bound to the collection itself.

Reply:
I think this does not count to premature abstraction as it increases readability if the abstraction levels of the calls are uniform. Yet this is open for discussion.